samples/skills/WriterSkill/Translate/config.json

```diff
@@ -3,7 +3,7 @@
   "type": "completion",
   "description": "Translate the input into a language of your choice",
   "completion": {
-    "max_tokens": 256,
+    "max_tokens": 2000,
```

Comment:
Why 2000 over 500, 1000 etc? I'm not aginst it, but it does make the skill less flexible to smaller models.

Reply:
For the example that I have, the translation of the output requires more tokens. Also, since this is `max_tokens`, it should still be okay for even simple translations right?

Reply:
Also had to rename the TranslateV2 skill because the Planner could get confused on which skill to use. Also TranslateV2 was mostly about improving English so renamed it to EnglishImprover